src/app/components/language-selector/language-selector.js

```diff
@@ -8,9 +8,12 @@ import './language-selector.scss';
 // You can't use a variable for id
```

Comment:
are you sure? I think we pass in a prop as the message ID in a few places on rex

Reply:
It works when I run it, but Jest fails
`    SyntaxError: /Users/rej2/Documents/GitHub/os-webview/src/app/components/language-selector/language-selector.js: [React Intl] Messages must be statically evaluate-able for extraction.
`

Reply:
I've reworked it to use Intl.DisplayNames for language names.